### PR TITLE
Contributing.md changes

### PR DESCRIPTION
**Pull request by mhegazy**
Updates to the contributing guidlines to allow new sample submission.

### CODE REVIEW

**Comment by mhegazy** [comment]
//cc @DanielRosenwasser, can you take a look.

**Comment by DanielRosenwasser** [comment]
I've committed some corrections. With those, :+1:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,9 +1,32 @@
...
-New samples will not be accepted at this time.
+## Contributing fixes to existing samples
+
+A bug must have an issue tracking it in the issue tracker that has been approved (labeled "help wanted") by the TypeScript team.
+Your pull request should include a link to the bug that you are fixing.
+If you've submitted a PR for a bug, please post a comment on the issue to avoid duplication of effort.
+
+## Contributing new samples
+
+New samples may be accepted, but will need to first be approved (marked "help wanted" by a TypeScript coordinator with the message "Approved") in the suggestion issue.
```
Comment (DanielRosenwasser) [inline CONTRIBUTING.md]: Why both "help wanted" and "Approved"?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
+
+## Contributing new samples
+
+New samples may be accepted, but will need to first be approved (marked "help wanted" by a TypeScript coordinator with the message "Approved") in the suggestion issue. 
+
+For new samples, please provide a detailed explanation of the intended sample, list of technologies or tools used, and an explanation of why existing samples are not sufficient and a new sample is needed.
+
+A new sample should:
+
+* include a detailed `README.md` file describing how to setup, build, and run the sample.
```
Comment (DanielRosenwasser) [inline CONTRIBUTING.md]: What about "include comments explaining why steps are being taken, including in build files if applicable."

**Comment by DanielRosenwasser** [comment]
:+1:

**Pull request merged**